packages/server/modules/automate/tests/trigger.spec.ts

```diff
@@ -1112,6 +1113,16 @@ const { FF_AUTOMATE_MODULE_ENABLED } = getFeatureFlags()
       let automationRun: InsertableAutomationRun
 
       before(async () => {
+        const testVersion = {
+          id: cryptoRandomString({ length: 10 }),
+          authorId: testUser.id,
+          streamId: testUserStream.id,
+          branchName: testUserStreamModel.name,
+          objectId: null
+        }
+
+        // @ts-expect-error force setting the objectId to null
+        await createTestCommit(testVersion)
```

Comment:
better to leave the objectId an empty string as instructed in the type, than to override a TS error